ntbs-service/Models/Validations/ValidationHelper.cs

```diff
@@ -61,16 +62,16 @@ public static string ValidYearLaterThanBirthYear(int birthYear)
         #endregion
 
         #region Contact Tracing
-        public const string ContactTracingAdultsScreened = "Must not be greater than the number of adults identified";
-        public const string ContactTracingChildrenScreened = "Must not be greater than the number of children identified";
-        public const string ContactTracingAdultsLatentTB = "Must not be greater than number of adults screened minus those with active TB";
-        public const string ContactTracingChildrenLatentTB = "Must not be greater than number of children screened minus those with active TB";
-        public const string ContactTracingAdultsActiveTB = "Must not be greater than number of adults screened minus those with latent TB";
-        public const string ContactTracingChildrenActiveTB = "Must not be greater than equal to than number of children screened minus those with latent TB";
-        public const string ContactTracingAdultStartedTreatment = "Must not be greater than number of adults with latent TB";
-        public const string ContactTracingChildrenStartedTreatment = "Must not be greater than number of children with latent TB";
-        public const string ContactTracingAdultsFinishedTreatment = "Must not be greater than number of adults the started treatment";
-        public const string ContactTracingChildrenFinishedTreatment = "Must not be greater than number of children the started treatment";
+        public const string ContactTracingAdultsScreened = "Adults screened must not be greater than the number of adults identified";
```

Comment:
Pretty much necessary for this feature - but I assume we checked the error message changes by Nancy?

Reply:
These will be run by Nancy but haven't been yet - will post the outcome of the chat on the PR